src/runtime_src/core/tools/common/tests/aie_pl_util/pl_controller_aie2.cpp

```diff
@@ -267,52 +238,5 @@ std::vector<tile_type> plController_aie2::get_tiles(const std::string& graph_nam
     return tiles;
 }
 
-std::vector<buffer_type> plController_aie2::get_buffers(const std::string& port_name) {
-    boost::property_tree::ptree dma_meta;
-    std::ifstream jsonFile(m_dma_info_path);
-    if (!jsonFile.good())
-	throw std::runtime_error("get_buffers():ERROR:No dma info file specified");
-
-    boost::property_tree::json_parser::read_json(m_dma_info_path, dma_meta);
-    std::vector<buffer_type> buffers;
-
-    for (auto& buffer : dma_meta.get_child("S2MM")) {
-        for (auto& node : buffer.second.get_child("KernelPort")) {
-            if (node.second.get_value<std::string>() != port_name) continue;
-            int count = 0;
-            for (auto& buff_info : buffer.second.get_child("BufferInfo")) {
-                for (auto& field : buff_info.second.get_child("BD")) {
-                    buffers.push_back(buffer_type());
-                    auto& b = buffers.at(count++);
-                    b.col = buff_info.second.get<uint16_t>("Column");
-                    b.row = buff_info.second.get<uint16_t>("Row");
-                    b.ch_num = buff_info.second.get<uint16_t>("Channel");
-                    b.lock_id = buff_info.second.get<uint16_t>("LockID");
-                    b.bd_num = static_cast<uint16_t>(std::stoul(field.second.data()));
-                    b.s2mm = true;
-                }
-            }
-        }
-    }
-    for (auto& buffer : dma_meta.get_child("MM2S")) {
-        for (auto& node : buffer.second.get_child("KernelPort")) {
-            if (node.second.get_value<std::string>() != port_name) continue;
-            int count = 0;
-            for (auto& buff_info : buffer.second.get_child("BufferInfo")) {
-                for (auto& field : buff_info.second.get_child("BD")) {
-                    buffers.push_back(buffer_type());
-                    auto& b = buffers.at(count++);
-                    b.col = buff_info.second.get<uint16_t>("Column");
-                    b.row = buff_info.second.get<uint16_t>("Row");
-                    b.ch_num = buff_info.second.get<uint16_t>("Channel");
-                    b.lock_id = buff_info.second.get<uint16_t>("LockID");
-                    b.bd_num = static_cast<uint16_t>(std::stoul(field.second.data()));
-                    b.s2mm = false;
-                }
-            }
-        }
-    }
-    return buffers;
-}
```

Comment:
Uses reference to the dma_lock_report.json file which is not relevant for aie2 pl test.